backend/fleet_management/tests/test_cars.py

```diff
@@ -0,0 +1,43 @@
+from urllib.parse import urlencode
+
+from django.contrib.auth.models import Group
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from fleet_management.constants import Groups
+from fleet_management.factories import CarFactory, UserFactory
+from fleet_management.serializers import CarSerializer
+
+
+class CarsApiTest(APITestCase):
+    def setUp(self):
+        self.url = reverse("cars")
+        self.user = UserFactory(username="Admin")
+        self.user.groups.set(Group.objects.filter(name=Groups.Driver.name))
+        self.cars = [
+            CarFactory(country=self.user.country, plates="BB73847KB"),
+            CarFactory(country=self.user.country, plates="FOO129338"),
+            CarFactory(country=self.user.country, plates="II43530TB"),
+        ]
+
+    def test_401_for_unlogged_user(self):
+        res = self.client.get(self.url)
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_get_all_cars(self):
+        self.client.force_login(self.user)
+        res = self.client.get(self.url)
+        cars = res.data
```

Comment:
You're using `data.json()` in `#40`th line, is there any reason that you've used `res.data` here instead?

Reply:
Actually no, it behaves the same just missed that.